packages/aws-cdk/lib/cli.ts

```diff
@@ -219,6 +220,7 @@ export async function exec(args: string[], synthesizer?: Synthesizer): Promise<n
 
       case 'ls':
       case 'list':
+        CliIoHost.currentAction = 'list';
```

Comment:
why only for these and not for other commands?

Reply:
simple answer is they are in phase 2 and no one is gonna be using them programmatically yet so it doesn't really matter